xblock/fields.py

```diff
@@ -233,6 +253,23 @@ def __eq__(self, other):
         return isinstance(other, Scope) and self.user == other.user and self.block == other.block
 
 
+class RemoteScope(Scope):
+    """
+    Defines types of remote scopes to be used.
+    """
+    shared_user_state = ScopeBase(SharedUserScope.ONE, BlockScope.USAGE, u'shared_user_state')
+    shared_group = ScopeBase(SharedUserScope.GROUP, BlockScope.USAGE, u'shared_group')
+    shared_platform = ScopeBase(SharedUserScope.ALL, BlockScope.USAGE, u'shared_platform')
```

Comment:
What is shared platform? It's not clear from the name. The way I see it, we want to probe across either user ids, usage ids or both?